core/vision/feed_handler.py

```diff
@@ -36,7 +36,7 @@ def feed_process(
             video_filename + "_left.avi",
             fourcc,
             frame_rate,
-            (resolution_x, resolution_y),
+            (1280, 720),
```

Comment:
While the resolution used here and line 48 appears to be an intentional departure and separation from the resolution_x and _y constants, I imagine this should still be defined as a constant somewhere.

Reply:
Yeah, this was changed to up resolution for recording purposes (SAR). Fixed.